src/metaswitch/clearwater/config_manager/alarms.py

```diff
@@ -45,5 +45,6 @@ def check_alarm(self):
         if all(self._files.values()):
             self._alarm.clear()
         else:
+            _log.warning("Config not in sync - status: " + str(self._files))
```

Comment:
Can the log text be clearer? E.g. 'At least one shared configuration file is out of sync. The status of the files is as follows (where the files marked as False are out of sync): <status>'

Reply:
Sorry I was being lazy. I've tweaked this line so it just logs the out of sync files - it'll now give:

```
30-01-2018 11:15:56.955 UTC WARNING alarms.py:52 (thread SproutSCSCFJsonPlugin): Config is not in sync - the following files are out of sync: ['/etc/clearwater/shared_ifcs.xml', '/etc/clearwater/shared_config']
30-01-2018 11:15:57.060 UTC WARNING alarms.py:52 (thread SharedIFCsXMLPlugin): Config is not in sync - the following files are out of sync: ['/etc/clearwater/shared_config']
```